src/controllers/users/register.ts

```diff
@@ -1,20 +1,28 @@
+import * as console from 'node:console'
+
 import { Request, Response } from 'express'
 
 import registerUserFromGoogle from '@sb/controllers/users/utils/register'
+import encrypt from '@sb/encryption/encrypt'
+import User from '@sb/interfaces/User'
 import { StatusCodes } from '@sb/types/status-codes'
 import logger from '@sb/utils/logger'
 
 
-async function register(request: Request, response: Response): Promise<void> {
-  const userInfo = request.body as Record<string, string>
+interface IncomingRegisterRequest extends Request {
+    body: Partial<User>
+}
 
+async function register(request: IncomingRegisterRequest, response: Response): Promise<void> {
   try {
-    const user = await registerUserFromGoogle(userInfo)
+    const user = await registerUserFromGoogle(request)
     if (user instanceof Error) {
+      console.log('a')
```

Comment:
This should have been picked up by the linter, remove and make sure console statements are included in linting rules